scene/gui/popup_menu.cpp

```diff
@@ -1800,7 +1805,7 @@ void PopupMenu::set_item_submenu(int p_idx, const String &p_submenu) {
 		}
 	}
 
-	items.write[p_idx].submenu = p_submenu;
+	items.write[p_idx].submenu = submenu_name_safe;
```

Comment:
Why are you assigning changed name if it's layer used for `get_node()`?

Reply:
Sorry, it was a mistake.